src/FrontDoor/FrontDoor/ChangeLog.md

```diff
@@ -19,6 +19,10 @@
 -->
 ## Upcoming Release
 
+## Version 1.6.0
+* Updated module to use API 2020-05-01.
+* Added Private link support for Storage, Keyvault and Web App Service resources.
```

Comment:
You don't need to update version. Our tool will handle this. The only section you need to update in changelog.md is `Upcoming Release`.